stylegan_code_finder/utils/augment_dataset.py

```diff
@@ -49,6 +49,7 @@ def augment_image(original_image: Image.Image, segmentation_image: Image.Image,
             iaa.GammaContrast((0.1, 1.0)),  # lighter
         ])),
         iaa.Sometimes(0.10, iaa.Invert()),
+        iaa.Sometimes(0.8, iaa.arithmetic.AdditiveGaussianNoise(scale=(30, 50))),
```

Comment:
0.8 feels like a lot, but I'm also not sure how large the impact is given the scale.
Hopefully, you visually verified the augmentation results.